pkg/util/compression/zstd.go

```diff
@@ -8,7 +8,7 @@
 package compression
 
 import (
-	"github.com/DataDog/zstd"
+	zstd "github.com/DataDog/zstd.v1.3"
```

Comment:
this should still use v0.5 of zstd, so should keep importing `github.com/DataDog/zstd` I believe

Reply:
I believe that this was previously using zstd 1.3 as that was under the _github.com/DataDog/zstd_ [previously](https://github.com/DataDog/datadog-agent/pull/3936/files#diff-836546cc53507f6b2d581088903b1785L10)

Reply:
sorry, for some reason I was convinced we needed to use a zstd version < 1.0 for this code to work well, disregard 😅